robottelo/ui/contentviews.py

```diff
@@ -30,6 +30,21 @@ def go_to_filter_page(self, cv_name, filter_name):
         strategy, value = locators['contentviews.select_filter_name']
         self.click((strategy, value % filter_name))
 
+    def set_calendar_date_value(self, name, value=None):
+        """Set the input value of a date field and press the button to hide
+         the calendar popup panel"""
+        date_element = self.wait_until_element(
+            locators['contentviews.calendar_date_input'] % name)
+        date_element.clear()
+        if value:
+            date_element.send_keys(value)
```

Comment:
Yeah, yesterday, I understood that we always wanted to clear that value, but if no and that is usual input, so we need to remove entire code and use `text_field_update` as I suggested from the start. And if user wanted to clear, he will send `''` and it will be cleared